mava/networks.py

```diff
@@ -159,6 +163,52 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> tfd.I
         )
 
 
+class RecQNetwork(nn.Module):
```

Comment:
Can we move this to the bottom of the file please so that it's next to the other recurrent network, just above the parse function